static/app/components/modals/sudoModal.tsx

```diff
@@ -130,6 +137,10 @@ class SudoModal extends React.Component<Props, State> {
     const user = ConfigStore.get('user');
     const isSelfHosted = ConfigStore.get('isSelfHosted');
 
+    if (errorType === ErrorCodes.invalidSSOSession) {
+      this.handleLogout();
+    }
```

Comment:
@maxiuyuan it'll be preferred if you `return null;` early here so that we can avoid any potential flickering that could occur when rendering React components and then calling `location.assign()`.

Reply:
to check my understanding here, are you saying to `return null;` after `this.handleLogout();` is called so the components below won't be rendered. Then call  `location.assign()` after the early return?
or 
just to `return null;` after `this.handleLogout()`?

Reply:
@maxiuyuan just `return null;` after `this.handleLogout()` on line 141. The motivation is so that the contents after line 142 will not render/execute.